server/ips.go

```diff
@@ -192,7 +193,7 @@ func getAllAvailableNetworks() ([]net.IPNet, error) {
 // that returns a reachable server's address to be used by local pairing client.
 func FindReachableAddressesForPairingClient(serverIps []net.IP) ([]net.IP, error) {
 	// TODO until we can resolve Android errors when calling net.Interfaces() just noop. Sorry Android
-	if runtime.GOOS == common.AndroidPlatform {
+	if strings.HasPrefix(runtime.GOOS, common.AndroidPlatform) {
```

Comment:
As discussed in dm, I found a way to suppress a single GoLand warning. 
Though it doesn't look pretty, I think with such solution the intentions are more clear. And we don't add any runtime operations.

```suggestion
	if //goland:noinspection GoBoolEpressions
	runtime.GOOS == common.AndroidPlatform {
```

Also, there one more place with such warning, could you please update it as well?

https://github.com/status-im/status-go/blob/6894295ac3b21a2739e68c7a8bce54f70a52854a/t/utils/utils.go#L225